test/system/030-run.bats

```diff
@@ -1465,4 +1465,34 @@ search               | $IMAGE           |
     is "$output" "Error.*: $expect" "podman emits useful diagnostic when no entrypoint is set"
 }
 
+@test "podman run - stopping loop" {
+    skip_if_remote "this doesn't work with with the REST service"
+
+    run_podman run -t -d --name testctr --stop-timeout 240 $IMAGE sh -c 'trap "echo READY" INT; while:; do sleep 0.1; done'
+
+    run_podman inspect --format '{{ .State.ConmonPid }}' testctr
+    conmon_pid=$output
+
+    $(PODMAN) stop testctr &
+    stop_pid=$!
+
+    while :; do
+	sleep 0.5
+	run_podman logs testctr
+	if [[ "$output" =~ READY ]]; then
+	    break
+	fi
+    done
```

Comment:
LGTM but since you have to repush anyway, please use `wait_for_ready testctr` instead of this loop